blogposts/2021/dev-tool-time-seth-vargo.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Dev Tool Time with Seth Vargo - April 14, 2021
```

Comment:
```suggestion
title: Dev Tool Time with Seth Vargo: Productivity hacks and .gitconfig tips
```

Reply:
I think the date of the episode is less important than giving readers an idea of what to expect from the post 😃